components/tournaments_listing/commons/tournaments_listing_conditions.lua

```diff
@@ -65,11 +65,15 @@ function TournamentsListingConditions.base(args)
 		conditions:add{ConditionNode(ColumnName('game'), Comparator.eq, args.game)}
 	end
 
-	if args.series then
-		conditions:add{ConditionTree(BooleanOperator.any):add{
-			ConditionNode(ColumnName('series'), Comparator.eq, args.series),
-			ConditionNode(ColumnName('extradata_series2'), Comparator.eq, args.series)
-		}}
+	if args.series1 or args.series then
+		local seriesConditions = ConditionTree(BooleanOperator.any)
+		for _, series in Table.iter.pairsByPrefix(args, 'series', {requireIndex = false}) do
```

Comment:
out of scope: the parameters for this module is very inconsistent. some are expected to be CSV, while others to use 1,2,3 suffix